azure-pipelines/build-release.yml

```diff
@@ -13,32 +13,32 @@ stages:
         Linux_Py_3_6:
           python.version: '3.6'
           vmImageName: ubuntu-18.04
-          uploadCoverage: "false"
+          triggerNonMatrixActions: "false"
```

Comment:
Pretty terrible name - would welcome suggestions. Something that indicates that this is what triggers a step that we only want to run once, not on every machine in the matrix. :s

Reply:
Maybe this is the wrong way to do it and we should have action specific attributes. eg. 
```
uploadCoverage: "true"
generateDocs: "true"
```

Reply:
I like this approach (2 actions) better, it is more explicit and gives us more flexibility if we ever need it. I guess this is where I become a YAGNI hypocrite.

_Shame on me for suggesting single flag._